### ISSUE TITLE
Handle individual WLED segments as isolated instances in Hyperion

### ISSUE THREAD

**Issue by pimw1** [hyperion-project/hyperion.ng#1437]
## Feature request
During recent development, WLED is able to handle multiple segments. This is a very nice and powerful feature if you bought the proper hardware for driving your leds, such as the the Quinled Dig Quad + ESP32 (https://quinled.info/pre-assembled-quinled-dig-quad/).

The beauty of different segments is that you can mix different ledstrips (for example combining ws2812b 60leds/meter with ws2812b 144leds/meter).

Hyperion is currently unable to handle those WLED segments as individual segments.

#### What problem does this feature solve?
Driving more fancy led-setups with Hyperion. Examples:
-driving ledstrips with different led-density
-combining ledstrips behind the tv with funny places ledstrips somewhere else in the room
-etc

The only way to do this currently, is by buying an individual ESP for every segment, since Hyperion will handle every ESP as an instance.

So in short: this feature will open a lot of awesome possibilites with WLED.

#### What does the proposed API look like?
I don't know exactly, since i'm lacking the technical knowlegde here. But i think this should be an extension of the current solution, in which you choose which segments to treat as individual instances and which instances to handle as a group instance.

#### How should this be implemented in your opinion?
Something like this:
![image](https://user-images.githubusercontent.com/74881023/156933490-f268de5d-f45f-41d8-8393-a347e6e89552.png)

This will give maximum flexibility in segment-assignment.

#### Are you willing to work on this yourself?
Yes, but i cannot programm. I can help in thinking about design, i can help in testing. And i can help in moral support ;)
